src/utils/findpackage/packageunix.rs

```diff
@@ -17,25 +18,43 @@ use super::{
 
 // here is the logic of findpackage on linux
 //
-const PREFIXS: [&str; 2] = ["/usr", "/usr/local"];
+static PREFIXES: LazyLock<Vec<String>> = LazyLock::new(|| {
```

Comment:
maybe should put this function in findpackage.rs, and use it in 

``packageunix.rs`, "packagewindows.rs", "packagemacos.rs"

and we also need to reserve the default prefix, and if there is the prefix by cmake, replace it with cmake one